src/Microsoft.ComponentDetection.Detectors/rust/RustCliDetector.cs

```diff
@@ -84,6 +84,13 @@ protected override async Task OnFileFoundAsync(ProcessRequest processRequest, ID
 
             var metadata = CargoMetadata.FromJson(cliResult.StdOut);
             var graph = BuildGraph(metadata);
+
+            var packages = metadata.Packages.ToDictionary(
+                x => $"{x.Name} {x.Version}",
+                x => (
+                    (x.Authors == null || x.Authors.All(a => string.IsNullOrWhiteSpace(a))) ? null : string.Join(", ", x.Authors),
```

Comment:
This could be changed to `(x.Authors == null || !x.Authors.Any(a => string.IsNullOrWhiteSpace(a))) ? null : string.Join(", ", x.Authors)` so it isn't always evaluating the whole array, but not strictly necessary since author arrays will have a limited size.